apis/v1beta1/types.go

```diff
@@ -41,6 +41,9 @@ type ProviderConfigSpec struct {
 	// Useful for AWS API implementations that do not have the IAM, STS API, or metadata API
 	// +optional
 	SkipReqAccountId bool `json:"skip_requesting_account_id,omitempty"`
+	// Ignore tags with provdied prefixes
+	// +optional
+	IgnoreTagsPrefixes []string `json:"ignore_tags_prefixes,omitempty"`
```

Comment:
The json should be in camelCase, to fit with existing kubernetes conventions. Unfortunately some of the parameters above were named in violation of that convention.

Reply:
Sure, I was also confused. I'll change that to camelCase.